src/applications/check-in/pre-check-in/pages/Error/tests/Error.unit.spec.jsx

```diff
@@ -45,16 +46,42 @@ describe('check-in', () => {
         );
         const dateMessage = component.getByTestId('date-message');
         expect(dateMessage).to.exist;
-        expect(
-          within(dateMessage).getByText(
-            'You can pre-check in online until 01/02/2022.',
```

Comment:
I like not having static dates in the tests but I kind of don't like losing the verification of the correct date calculation - do we have a unit test for that elsewhere? If so I'm good with this 👍

Reply:
So the issue here was that since the test date was in the past, it would always show the expired "15 minutes past appointment" error condition, which would cause this test to fail. Any ideas on how to divest the date from that issue so it could be tested elsewhere?

Reply:
One possibility I can think of is to mock the system date/time like [we're doing here](https://github.com/department-of-veterans-affairs/vets-website/blob/f56f964a17a4aafd50715fbacaf490f9e96ae4b1/src/applications/check-in/utils/navigation/navigation.unit.spec.js#L77), another possibility might be to pull out the code that generates that date into a util function and unit test that function. I'm open to whatever 🙂

Reply:
the mocked date worked great! Updated.